CODEOWNERS

```diff
@@ -76,8 +76,8 @@ go.mod @fleetdm/go
 #
 # (see website/config/custom.js for DRIs of other paths not listed here)
 ##############################################################################################
-/docs                                           @rachaelshaw
-/docs/Using-Fleet/REST-API.md                   @rachaelshaw # « REST API reference documentation
+/docs                                           @rachaelshaw @eashaw @noahtalerman
```

Comment:
When something needs to be reviewed, 1 person can hear about it and review it. 

I.e. 1 DRI not 3

Reply:
@mikermcneil This is to provide backups while @rachaelshaw is out.

Reply:
Let's go with @eashaw